modules/steam/hm.nix

```diff
@@ -0,0 +1,60 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.stylix.targets.steam;
+
+  adwaitaCustomCss = config.lib.stylix.colors {
+    template = ./custom.mustache;
+    extension = "css";
+  };
+
+  adwaitaTheme = pkgs.stdenv.mkDerivation (_self: {
+    name = "Adwaita-for-Steam";
+    version = "3.1";
```

Comment:
This version field is very likely to become out-dated. It might be better to remove it:

```suggestion
```